common/djangoapps/student/admin.py

```diff
@@ -2,6 +2,9 @@
 from django import forms
 from django.contrib.auth import get_user_model
 from django.contrib.auth.admin import UserAdmin as BaseUserAdmin
+from django.contrib.auth.forms import UserCreationForm as BaseUserCreationForm
+from django.contrib.auth.forms import UserChangeForm as BaseUserChangeForm
+from django.core.exceptions import ValidationError
```

Comment:
it's not necessary to import it directly from exceptions, you can use `forms. ValidationError` with the same parameters

Reply:
ok, done